framework/test/mocha/unit/modules/chain/blocks/block_version.js

```diff
@@ -159,18 +219,18 @@ describe('block_version', () => {
 				).to.equal(false);
 			});
 
-			it('should return true for version = 1, height = 102', async () => {
+			it('should return true for version = 2, height = 102', async () => {
 				const height = 102;
-				const version = 1;
+				const version = 2;
 
 				return expect(
 					blockVersion.isValid(version, height, exceptions)
 				).to.equal(true);
```

Comment:
I don't understand why it's returning true when the exceptions are not present at all?

Reply:
Can you double check these tests?

Reply:
For this test, there is one exception present and the current block version is set to 2.
So it returns true for these tests.

Reply:
Currently `Currentblockversion = 2` so the test are returning true correctly.